src/any_agent/tracing/trace.py

```diff
@@ -148,6 +149,29 @@ class AgentTrace(BaseModel):
 
     model_config = ConfigDict(arbitrary_types_allowed=True)
 
+    @property
+    def execution_time(self) -> timedelta:
+        """Returns the execution time of the AGENT span named 'any_agent' as a datetime.timedelta object.
+
+        The duration is computed from the span's start and end time (in nanoseconds).
```

Comment:
Note that "execution time", in a multicore world with several processor clocks, can be misleading :) Maybe "request time" or "request duration" would be a more suitable name.

Reply:
Ooo. Let me rename this to trace_duration, I think that makes the meaning clearer.